### DISCUSSION TITLE
blog/release-of-tailwind-nextjs-starter-blog-v2

### DISCUSSION THREAD

**Discussion by giscus** [mrlopezco/lopezconextjs#2] [General]
# blog/release-of-tailwind-nextjs-starter-blog-v2

Release of Tailwind Nextjs Starter Blog template v2.0, refactored with Nextjs App directory and React Server Components setup.Discover the new features and how to migrate from V1.

http://localhost:3000/blog/release-of-tailwind-nextjs-starter-blog-v2.0

**Comment by mrlopezco** [comment]
This is a test
